metadata-ingestion/src/datahub/ingestion/api/ingestion_state_provider.py

```diff
@@ -1,31 +0,0 @@
-from abc import ABC, abstractmethod
-from typing import Any, Dict, NewType, Optional
-
-from datahub.ingestion.api.common import PipelineContext
-from datahub.metadata.schema_classes import DatahubIngestionCheckpointClass
-
-JobId = NewType("JobId", str)
-
-
-class IngestionStateProvider(ABC):
```

Comment:
@rslanka : can you confirm that this is safe to remove?

Reply:
Yes, this is safe to remove. This got replaced by [ingestion_job_state_provider.py](https://github.com/datahub-project/datahub/blob/master/metadata-ingestion/src/datahub/ingestion/api/ingestion_job_state_provider.py).